cogs/sudo.py

```diff
@@ -6,7 +6,9 @@
 import psutil
 from discord import Color, Embed
 from discord import __version__ as discord_version
-from discord.ext.commands import Bot, Cog, Context, group
+from discord.ext.commands import Bot, Cog, Context, check, group
+import typing as t
+from .utils import constants
```

Comment:
Please sort all imports, vscode has a built-in way to do this:
![Peek 2020-06-16 10-55](https://user-images.githubusercontent.com/20902250/84753944-f3ac6f80-afbf-11ea-8e64-ed7d0f8afdec.gif)